### PR TITLE
nrf_cc3xx v0.9.7

### PR DESCRIPTION
**Pull request by frkv**
-Adding version 0.9.7 of CryptoCell runtime library
-Fixing issue with memory slab indirection in nrf_cc3xx_platform RTOS-
 files
-Adding changelog items
-Setting 0.9.7 as selected version in crypto/CMakeLists.txt

https://github.com/nrfconnect/sdk-nrf/pull/3781

ref: NCSDK-7915
ref: NCSDK-7914
ref: NCSDK-7910
ref: NCSDK-7856

Signed-off-by: Frank Audun Kvamtrø <frank.kvamtro@nordicsemi.no>

### CODE REVIEW

File: crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst [hunk 1]
```diff
@@ -23,6 +25,86 @@ Added
 
 Added new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
 
+.. note:
```
Comment (Vge0rge) [inline crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst:28]:
```suggestion
.. note::
```

File: crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst [hunk 2]
```diff
@@ -23,6 +25,86 @@ Added
...
+
+New version of the runtime library fixing a regression in derived keys for ECB, CCM and GCM
+Library built against mbed TLS version 2.24.0
+
+This version is dependent on the nrf_cc310_platform or nrf_cc312_platform library for low-level initialization of the system and proper RTOS integration.
+
+Added
+=====
+
+Added new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
```
Comment (Vge0rge) [inline crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst:106]:
```suggestion
Added new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840 and nRF5340.
```

File: crypto/nrf_cc310_platform/CHANGELOG.rst [hunk 3]
```diff
@@ -9,12 +9,91 @@ Changelog - nrf_cc3xx_platform
...
+New version of the library with a bugfix:
+* Fixed an issue with mutex slab allocation in zephyr RTOS platform file.
 
-Library built against mbed TLS version 2.24.0.
+Library built against mbed TLS version 2.24.0
+
+Added
+=====
+
+Added new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
```
Comment (Vge0rge) [inline crypto/nrf_cc310_platform/CHANGELOG.rst:23]:
```suggestion
Added new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840 and nRF5340.
```

**Review by Vge0rge — APPROVED** [review]

**Review by torsteingrindvik — APPROVED** [review]

File: crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst [hunk 4]
```diff
@@ -23,6 +25,86 @@ Added
...
+  * short-wchar, no interrupts
+
+    * ``cortex-m4/soft-float/short-wchar/no-interrupts/libnrf_cc310_mbedcrypto_0.9.7.a``
+    * ``cortex-m4/hard-float/short-wchar/no-interrupts/libnrf_cc310_mbedcrypto_0.9.7.a``
+
+
+nrf_cc3xx_mbedcrypto - 0.9.6
+****************************
+
+New version of the runtime library fixing a regression in derived keys for ECB, CCM and GCM
```
Comment (b-gent) [inline crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst:98]:
```suggestion
New version of the runtime library fixing a regression in derived keys for ECB, CCM, and GCM.
```

File: crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst [hunk 5]
```diff
@@ -23,6 +25,86 @@ Added
...
+Library built against mbed TLS version 2.24.0
```
Comment (b-gent) [inline crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst:99]:
```suggestion
Library built against mbed TLS version 2.24.0.
```

**Review by b-gent — APPROVED** [review]
Adding comments with some nits that could be fixed, unless this is urgent to get in.

File: crypto/nrf_cc310_platform/CHANGELOG.rst [hunk 6]
```diff
@@ -9,12 +9,91 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
-nrf_cc3xx_platform - 0.9.6
+nrf_cc3xx_platform - 0.9.7
 **************************
 
-New version of the library with mbed TLS sources ctr_drbg.c and entropy.c built in.
+New version of the library with a bugfix:
+* Fixed an issue with mutex slab allocation in zephyr RTOS platform file.
```
Comment (b-gent) [inline crypto/nrf_cc310_platform/CHANGELOG.rst:16]:
```suggestion
* Fixed an issue with mutex slab allocation in Zephyr RTOS platform file.
```

File: crypto/nrf_cc310_platform/CHANGELOG.rst (hunk 3 above)
Comment (b-gent) [inline crypto/nrf_cc310_platform/CHANGELOG.rst:23]:
```suggestion
Added a new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
```

**Review by b-gent — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]
Checked for symbols

**Review by magnev — APPROVED** [review]

**Pull request merged**